packages/components/src/components/dialog/dialog.tsx

```diff
@@ -0,0 +1,107 @@
+import * as DialogPrimitive from '@radix-ui/react-alert-dialog';
+import { ComponentPropsWithoutRef, ElementRef, forwardRef } from 'react';
+
+import { cn } from '~/lib/utils';
+
+const Dialog = DialogPrimitive.Root;
+
+const DialogTrigger = forwardRef<
+  ElementRef<typeof DialogPrimitive.Trigger>,
+  ComponentPropsWithoutRef<typeof DialogPrimitive.Trigger>
+>(({ className, ...props }, ref) => (
+  <DialogPrimitive.Trigger className={cn('', className)} ref={ref} {...props} />
```

Comment:
```suggestion
  <DialogPrimitive.Trigger className={cn(className)} ref={ref} {...props} />
```

Reply:
I aimed to check if I need extra styles in a few places and forgot to clean it up. Thanks for catching this, Jorge🙏